### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by jorgesumle** [mquinson/po4a-website#8]
There isn't any license in this repository. I recommend publishing it under a free software license.

**Comment by mquinson** [comment]
Yes of course. It is intended to be released under the same license than po4a itself. Sorry for not being more explicit, I'll fix it soon.
